### DISCUSSION TITLE
Document how to install as a nixpkg / flake?

### DISCUSSION THREAD

**Discussion by bhipple** [ananthakumaran/paisa#83] [Q&A]
It looks like you're already using nix flakes for all of your development.  Is there a way to build the whole project with `nix build` and run it that way?  If so it would be nice if you could document it in https://paisa.fyi/getting-started/installation/

**Comment by ananthakumaran** [comment]
I use only the dev-shell/nix-shell, I haven't figured out how to package npm together with go. You can clone the project, get into the nix-shell and run `make install`. I like to provide a flake package, but I have yet to figure out the details.

**Reply by ananthakumaran** [reply]
@bhipple I have pushed some changes to the flake branch, so you might be able to build the cli via `nix build github:ananthakumaran/paisa/flake`. I am still figuring out the details, only cli is available now. Let me know if it works for you. Make sure you have `ledger` available on your $PATH

**Comment by bhipple** [comment]
Yes, that worked seamlessly.  Thank you!  Seems worth adding to master and the readme for others to enjoy as well?

**Reply by ananthakumaran** [reply]
Thanks for the confirmation, will do that.
